cli/README.md

```diff
@@ -15,7 +15,7 @@ $ solana-foundation-delegation-program --version
 
 To register you will need:
 1. Two validator identity keypairs; one for the Solana Testnet and another for the Solana Mainnet
-2. Approximately ◎0.002 to submit the registration
+2. Approximately ◎0.002 on your testnet validator account (on mainnet network) to submit the registration
```

Comment:
This isn't quite accurate, the SOL can be in any mainnet account that you control

Reply:
Really? Because first of all I had sent to my Testnet Validator account on the mainnet network but I got the following error:

```
$ solana -um balance
0 SOL

$ solana-foundation-delegation-program apply --mainnet mainnet-validator-keypair.json --testnet validator-keypair.json --confirm
Mainnet Validator Identity: 8uoLiaHnXd3cZL96UwfWYVVtBuwBYYzMEGMmtLn115mS
Testnet Validator Identity: 4UNcH9sxWUo6bfZY93gmPiGZNssEgmG9Ho7C9ecjMv5N
Error: "4UNcH9sxWUo6bfZY93gmPiGZNssEgmG9Ho7C9ecjMv5N has insufficient balance. ◎0.001358280 required"
```
I had to execute the following command in order to have the funds on the right account 
```solana transfer --from ~/mainnet-validator-keypair.json 4UNcH9sxWUo6bfZY93gmPiGZNssEgmG9Ho7C9ecjMv5N 0.002 --allow-unfunded-recipient -um --fee-payer ~/mainnet-validator-keypair.json```

Reply:
It looks like your testnet validator account was just set as the default keypair.  `solana config set --keypair ...` can be used to change the default.

Alternatively, `solana-foundation-delegation-program --keypair mainnet-validator-keypair.json apply --mainnet mainnet-validator-keypair.json --testnet validator-keypair.json --confirm` would have worked as well.